src/app/pages/home/home.component.html

```diff
@@ -1,5 +1,57 @@
 <div class="page-header" data-parallax="true">
   <div class="filter"></div>
+
+  <nav class="navbar navbar-expand-lg navbar-dark">
```

Comment:
@Jaman-dedy  the second navigation is coming in here, can we remove it and maintain the singular existent application navigation

Reply:
Thanks @Rhytah , I worked on that